cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformParameters.java

```diff
@@ -275,16 +277,28 @@ private Map<PolicyType, String> initCdpRangerAuditS3PolicyJson() {
     }
 
     private Map<PolicyType, String> getPolicyJson(String resourceDefinition) {
+        return getPolicyJson(resourceDefinition, Optional.empty());
+    }
+
+    private Map<PolicyType, String> getPolicyJson(String resourceDefinition, Optional<String> govResourceDefinition) {
         String minified = JsonUtil.minify(resourceDefinition);
-        if (JsonUtil.INVALID_JSON_CONTENT.equals(minified)) {
-            String message = String.format("Cannot initialize Cloudbreak's policies JSON for AWS: %s", minified);
+        String govMinified = minified;
+        if (govResourceDefinition.isPresent()) {
+            govMinified = JsonUtil.minify(govResourceDefinition.get());
+        }
+        if (JsonUtil.INVALID_JSON_CONTENT.equals(minified) || JsonUtil.INVALID_JSON_CONTENT.equals(govMinified)) {
+            String message = String.format("Cannot initialize Cloudbreak's policies JSON for AWS: %s and AWS_GOV: %s", minified, govMinified);
             LOGGER.info(message);
             throw new CloudConnectorException(message);
         }
         Map<PolicyType, String> policy = new HashMap<>();
-        policy.put(PolicyType.GOV, Base64.encodeBase64String(minified.replaceAll(":aws:", ":aws-us-gov:").getBytes()));
+        policy.put(PolicyType.GOV, Base64.encodeBase64String(tranformToGovCloud(govMinified).getBytes()));
         policy.put(PolicyType.PUBLIC, Base64.encodeBase64String(minified.getBytes()));
         return policy;
     }
 
+    private String tranformToGovCloud(String minified) {
```

Comment:
Having separeted json it seems to be unnecessary.

Reply:
just for the minimal policy we have separated json so not for all of them